Update setting custom target name doc (#1789)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md

> To set a custom target name for a job in dbt Cloud, configure the "Target Name" field for your job in the Job Settings page.

Reviewer: ```suggestion
To set a custom target name for a job in dbt Cloud, configure the **Target Name** field for your job in the Job Settings page.
```

Reviewer: sugg: bold UI text field names

<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/jobs-settings-target-name.png" title="Overriding the target name to 'prod'"/>

## dbt Cloud IDE

> When developing in dbt Cloud, you can set a custom target name in your development credentials. Head to your account (via the gear in the top right hand corner), select the project under "Credentials" and update the target name.

Reviewer: ```suggestion
When developing in dbt Cloud, you can set a custom target name in your development credentials. Go to your account (from the gear menu in the top right hand corner), select the project under **Credentials**, and update the target name.
```

Reviewer: suggestion: minor nits for more clarity like removing potential USAn colloquialism (head), avoid Latin abbreviation, and used an Oxford comma in the series
